.circleci/config.yml

```diff
@@ -40,6 +40,7 @@ commands:
           command: |
             . /opt/conda/etc/profile.d/conda.sh
             conda activate esmvaltool
+            mamba --version
```

Comment:
It would be nicer to log this to the version logging step:
https://github.com/ESMValGroup/ESMValTool/blob/cd99fbea23e1ea2eb56c0b09fc4fb6df9cc35c2a/.circleci/config.yml#L24
and maybe use something like `conda env export --name base > base_environment.yml` to log the complete base environment for easier reproducibility?

Reply:
good idea! Just did :+1:

Reply:
And remove the logging from here?

Reply:
what logging? :log:

Reply:
Sorry if that was not clear. I meant, remove the line `mamba --version` since that version is now logged as part of the `base` environment export.

Reply:
the emoji was a (failed, wooden) log, since I can't post GH emojis in this antiquated browser, unless I literally type it like :-r-o-f-l-: :rofl:

Reply:
It's creating noise and maintainance if you ask me, but it's up to you :beers:

Reply:
ez view, bud - I likes it :grin: Cheers for reviewing :beer: